contracts/governance/extensions/GovernorExtendedVoting.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts vX.X.X (governance/extensions/GovernorExtendedVoting.sol)
+
+pragma solidity ^0.8.0;
+
+import "../Governor.sol";
+import "../../utils/math/Math.sol";
+
+/**
+ * @dev Extension of {Governor} for extending voting past quorum being reached.
+ *
+ * _Available since v4.X._
+ */
+abstract contract GovernorExtendedVoting is Governor {
+    using SafeCast for uint256;
+    using Timers for Timers.BlockNumber;
+
+    uint64 private _votingDelayExtension;
```

Comment:
"Delay" is not the right term. `votingDelay` measures delay between proposal creation and start of the vote.

The voting "period" what is extended by this module.